.github/workflows/main.yml

```diff
@@ -193,6 +193,7 @@ jobs:
     - name: CMake Generate
       shell: bash
       run: |
+        EXTENDED_BUILD_CONFIG=""
```

Comment:
Small fix: This was not initialized so would error out if none of the "extended" conditions were set.